### ISSUE TITLE
[Misc] Repository instructions point to the AWS China documentation

### ISSUE THREAD

**Issue by spensireli** [awslabs/service-workbench-on-aws#1134]
The README.md makes reference to the documentation for the installation of AWS Service Workbench. This documentation redirects you to the AWS China implementation of AWS Service Workbench. I would think you would want to have it to the non "cn" repository for the installation instructions for the majority of countries.

**Comment by maghirardelli** [comment]
Hi @spensireli! Thanks for the input!

Could you share some examples of where the "cn" links are? Also, if you are able, feel free to open a PR replacing them with non "cn" documentation to fix this!

Thanks,
Marianna

**Comment by maghirardelli** [comment]
Hi @spensireli! I actually found what you were referring too. Those were actually links to the [SWB extension to be CN compatible repo](https://github.com/awslabs/service-workbench-on-aws-cn ). We've removed them, in favor of our new [Implementation Guide](https://docs.aws.amazon.com/solutions/latest/service-workbench-on-aws/solution-overview.html) which was just published this week in an effort to standardize and centralize our documentation. Thanks for pointing this out to us. I'm going to close this issue as the links have been changed.
